test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php

```diff
@@ -164,7 +164,14 @@ static function (string $symbol) : array {
             array_filter(
                 $allSymbols,
                 static function (string $symbol) : bool {
-                    return (new CoreReflectionClass($symbol))->isInternal();
+                    $reflection = new CoreReflectionClass($symbol);
+
+                    if (! $reflection->isInternal()) {
+                        return false;
+                    }
+
+                    // Some extensions are broken
```

Comment:
Might be sensible to create an issue for these to revisit when they're working again, and add the issue link here?

Reply:
Yes, I plan to revisit it :)

Reply:
yes, but an issue ID is needed in this TODO

Reply:
Added.